src/controllers/mainController.ts

```diff
@@ -1502,9 +1502,13 @@ export default class MainController implements vscode.Disposable {
         });
 
         if (this.isRichExperiencesEnabled) {
+            // Register the command as async and forward all arguments
             this._context.subscriptions.push(
-                vscode.commands.registerCommand(Constants.cmdSchemaCompare, async (node: any) =>
-                    this.onSchemaCompare(node),
+                vscode.commands.registerCommand(
+                    Constants.cmdSchemaCompare,
+                    async (...args: any[]) => {
```

Comment:
Can you please add proper types here?

Reply:
properties datatypes would be any of these

ConnectionNode | TreeNodeInfo | SchemaCompareEndpointInfo | boolean | undefined